### DISCUSSION TITLE
Listener not show result

### DISCUSSION THREAD

**Discussion by uluumbch** [orchidsoftware/platform#2719] [Q&A] [answered]
So I try to make listener, but my creation was not show result after I type input.
Then I am trying to follow tutorial from docs [https://orchid.software/en/docs/listener/](https://orchid.software/en/docs/listener/) but it result the same, it show nothing after I fill the input. Do I doing it wrong?

The console reporting error `undefined array key 0`
![image](https://github.com/orchidsoftware/platform/assets/49960993/0b4f139e-08ac-4b57-a03e-cb3ddaa10e61)

![image](https://github.com/orchidsoftware/platform/assets/49960993/5b893d37-7fcc-4a47-a3fa-53c34c18a576)

**Comment by uluumbch** [comment, marked as answer]
nevermind guys, I think the docs have some issue, I just change the `handle` function 
before
```
 public function handle(Repository $repository, Request $request): Repository
    {
        [$minuend, $subtrahend] = $request->all();

        return $repository
            ->set('minuend', $minuend)
            ->set('subtrahend', $subtrahend)
            ->set('result', $minuend - $subtrahend);
    }
```

after
```
 public function handle(Repository $repository, Request $request): Repository
    {
        $minuend = $request->get('minuend', 0);
        $subtrahend = $request->get('subtrahend', 0);

        return $repository
            ->set('minuend', $minuend)
            ->set('subtrahend', $subtrahend)
            ->set('result', $minuend - $subtrahend);
    }
```

**Reply by RuLeiFei** [reply]
Thank you! You saved my day!
